### PR TITLE
Release 1.6.2

### PR DESCRIPTION
**Pull request by Dean-Coakley**
### Proposed changes
* Update NGINX version `1.17.7` -> `1.17.8`
* Fix docs: Order of commands, typos.
* Release 1.6.2

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs-web/installation/building-ingress-controller-image.md [hunk 1]
```diff
@@ -22,17 +22,17 @@ We build the image using the make utility and the provided `Makefile`. Let’s c
...
 1. Build the image:
     * For NGINX:
       ```
-      $ make clean
       $ make PREFIX=myregistry.example.com/nginx-ingress
       ```
       `myregistry.example.com/nginx-ingress` defines the repo in your private registry where the image will be pushed. Substitute that value with the repo in your private registry.
-      
+
       As the result, the image **myregistry.example.com/nginx-ingress:1.6.0** is built and pushed to the registry. Note that the tag `1.6.0` comes from the `VERSION` variable, defined in the Makefile.
```
Comment (Rulox) [inline docs-web/installation/building-ingress-controller-image.md]: Do we change 1.6.0 here? Not sure if we did this before 🤔

Comment (Dean-Coakley) [reply docs-web/installation/building-ingress-controller-image.md]: Hmm. We didn't before, but it should be changed.

File: docs-web/installation/building-ingress-controller-image.md [hunk 2]
```diff
@@ -22,17 +22,17 @@ We build the image using the make utility and the provided `Makefile`. Let’s c
...
     $ git clone https://github.com/nginxinc/kubernetes-ingress/
-    $ git checkout v1.6.1
+    $ cd kubernetes-ingress
+    $ git checkout v1.6.2
     ```
 
 1. Build the image:
     * For NGINX:
       ```
-      $ make clean
```
Comment (pleshakov) [inline docs-web/installation/building-ingress-controller-image.md]: no need to delete make clean. Changes to the Makefile have been made to the edge version, not release version.
same in the commands for Plus

**Comment by pleshakov** [comment]
@Dean-Coakley https://github.com/nginxinc/kubernetes-ingress/blob/ead56a090ffc84c6a00351ccdf87f63d9db097b3/docs-web/third-party-modules/opentracing.md :(

**Pull request merged**